hw/ip/aes/pre_sca/alma_post_syn/load_env.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# This file needs to be sources into shell as follows:
+# `source load_env.sh`
+
+# The top folder for opentitan repository
+export REPO_TOP=~/Documents/opentitan
+# The top folder for the private ASIC repo
+export ASIC_TOP=~/Documents/asic
+# The top folder for the CocoAlma repo
+export ALMA_TOP=~/Documents/coco-alma
```

Comment:
Would it be better to derive these variables from the `REPO_TOP` variable? I am not sure whether it's better to assume stuff lies under `~/Documents` or to use relative paths. In any case, I think the script should check that these directories indeed exist and exit otherwise.

Reply:
Thanks, that's great.